src/index.html

```diff
@@ -2,13 +2,59 @@
 <html lang="en">
   <head>
     <meta charset="UTF-8">
-    <meta name="viewport"
-          content="width=device-width, user-scalable=no, initial-scale=1.0, maximum-scale=1.0, minimum-scale=1.0">
+    <meta
+      name="viewport"
+      content="width=device-width, user-scalable=no, initial-scale=1.0,
+               maximum-scale=1.0, minimum-scale=1.0"
+    >
     <meta http-equiv="X-UA-Compatible" content="ie=edge">
     <title>Moyo header</title>
-    <link rel="stylesheet" href="./style.css">
+    <link
+      href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap"
+      rel="stylesheet"
+    >
+    <link href="./style.css" rel="stylesheet" >
   </head>
   <body>
-    <h1>Moyo header</h1>
+    <form
+      action="https://github.com/Dimoshua/layout_layout_moyo-header.git"
+      method="post"
+    >
+      <header class="container">
+        <a class="logo" href="#home"></a>
+        <nav class="nav">
+          <ul class="nav__list is-active">
+            <a class="box is-active" href="#apple">Apple</a>
+          </ul>
+          <ul class="nav__list">
+            <a class="box" href="#samsung">Samsung</a>
+          </ul>
+          <ul class="nav__list">
+            <a class="box" href="#smartphones">Smartphones</a>
+          </ul>
+          <ul class="nav__list" >
+            <a
+              class="box"
+              data-qa="hover"
+              href="#laptops&computers"
+            >
+              Laptops&nbsp&amp&nbspComputers
+            </a>
+          </ul>
+          <ul class="nav__list">
+            <a class="box" href="#gadgets">Gadgets</a>
+          </ul>
+          <ul class="nav__list">
+            <a class="box" href="#tablets">Tablets</a>
+          </ul>
+          <ul class="nav__list">
+            <a class="box" href="#photo">Photo</a>
+          </ul>
+          <ul class="nav__list">
+            <a class="box" href="#video">Video</a>
+          </ul>
+        </nav>
```

Comment:
`ul `- it is a list,` li` - list item. you should use one `ul class='nav__list' `tag and `li class='nav__item'`s for each link